RIGS/migrations/0036_profile_is_approved.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.13 on 2020-01-10 14:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('RIGS', '0035_auto_20191124_1319'),
+    ]
+
+    operations = [
```

Comment:
Might be worth squishing these migrations together?

Reply:
One of these is a schema migration so needs to remain separate as far as I know. But I can squash the two model migrations